pages/contact.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div id="main-content" style="display: flex; justify-content: center; align-items: center;" >
+    <iframe  src="https://forms.office.com/Pages/ResponsePage.aspx?id=sAafLmkWiUWHiRCgaTTcYTCImEcjxnNCsC9qmpDkQshUQjM4RzdYMkNTOVVONjVDRFFTVTZCOUVSTy4u&embed=true" frameborder="0" marginwidth="0" marginheight="0" style="border: none; max-width:100%; max-height:100vh; width: 90%; height: 80vh" allowfullscreen webkitallowfullscreen mozallowfullscreen msallowfullscreen> </iframe>
```

Comment:
I think the height: 80vh is causing there to be a gap at the bottom of the page. Should this be there?

Reply:
done, have changed height

Reply:
Thanks Ruby. Looks good now